packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec

```diff
@@ -1,66 +1,98 @@
+# template: smart_proxy_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_datadir:%global _root_datadir %{_datadir}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
 %global gem_name smart_proxy_ansible
+%global plugin_name ansible
 
-%global foreman_proxy_dir /usr/share/foreman-proxy
+%global foreman_proxy_min_version 1.24
+%global foreman_proxy_dir %{_root_datadir}/foreman-proxy
 %global foreman_proxy_statedir %{_localstatedir}/lib/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
-%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
-%global smart_proxy_dynflow_bundlerd_dir %{?rhel:/opt/theforeman/tfm/root/}%{_datadir}/smart_proxy_dynflow_core/bundler.d
+%global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
+%global smart_proxy_dynflow_bundlerd_dir %{_datadir}/smart_proxy_dynflow_core/bundler.d
 
-Summary: Ansible support for Foreman smart proxy
-Name: rubygem-%{gem_name}
+Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 3.0.1
-Release: 1%{?foremandist}%{?dist}
-Group: Applications/System
-License: GPLv3
+Release: 2%{?foremandist}%{?dist}
+Summary: Smart-Proxy Ansible plugin
+Group: Applications/Internet
+License: GPL-3.0
```

Comment:
```suggestion
License: GPLv3
```

Reply:
These are annoying.

Reply:
I've been thinking about a translation table but saw a presentation from a fellow RHer today about https://framagit.org/upt/upt and I want to do a proof-of-concept to see if we can switch over from a lot of our custom tooling.